app/controllers/reservations_controller.rb

```diff
@@ -0,0 +1,30 @@
+class ReservationsController < ApplicationController
+  # POST /cooking_sessions/5/reservations
+
+
+  # POST /cooking_sessions/:cooking_session_id/reservations
+  # params => {cooking_session_id: 5}
+  def create
+    @cooking_session = CookingSession.find(params[:cooking_session_id])
```

Comment:
[:id] should work instead of [:cooking_session_id]?

Reply:
It needs to be cooking_session_id